modules/openy_features/openy_node/modules/openy_node_mbrshp/config/install/core.entity_form_display.node.membership.default.yml

```diff
@@ -6,9 +6,11 @@ dependencies:
     - field.field.node.membership.field_mbrshp_description
     - field.field.node.membership.field_mbrshp_image
     - field.field.node.membership.field_mbrshp_info
+    - field.field.node.membership.field_meta_tags
```

Comment:
I'm not sure that this CT needs meta tags. There is no individual page for it. It's available only in membership calculator paragraph and always rendered as teaser.